test/functional/feature_fee_estimation.py

```diff
@@ -383,6 +401,79 @@ def test_acceptstalefeeestimates_option(self):
         assert_equal(self.nodes[0].estimatesmartfee(1)["feerate"], fee_rate)
 
 
+    def send_and_mine_child_tx(self, broadcaster, miner, parent_tx, feerate):
+        u = {"txid": parent_tx["txid"], "vout": 0, "value": Decimal(parent_tx["tx"].vout[0].nValue) / COIN}
```

Comment:
`Decimal(parent_tx["tx"].vout[0].nValue) / COIN`

Is there not a `satToBtc()` already? WDYT about introducing one? I've noticed this conversion in this diff thrice.

Reply:
Good idea for a follow-up @rkrux

Reply:
yeah, would recommend this be a separate PR, since there are likely multiple instances in other tests where we could replace the satToBtc conversion with a utility function in the test framework.

Reply:
Yeah, I can pick this up in a separate PR.

Reply:
Quite a late reply here but I don't think I will get to this anytime soon, created an issue here for anyone else to pick up: https://github.com/bitcoin/bitcoin/issues/31345